superset/models/core.py

```diff
@@ -866,7 +866,10 @@ def _log_query(sql):
                 for sql in sqls[:-1]:
                     _log_query(sql)
                     self.db_engine_spec.execute(cursor, sql)
-                    cursor.fetchall()
+                    try:
+                        cursor.fetchall()
+                    except BaseException:
+                        pass
```

Comment:
Hmm, I might be missing something obvious here, but why are we calling `fetchall()` if we're not even storing the return value anywhere? Wouldn't it be enough to just call `exexute()` here and remove the fetch all together?

Reply:
I was confused about here too. We only want data from the last query.

Reply:
Aah yes, it's explained here: #5517. I think this looks good, but it might be good to add a comment explaining that some engines require calling `fetchall()` to force the `execute()` calls to execute synchronously.

Reply:
It seems like the raw connection exposes the underlying DB-API. Per [PEP 249](https://www.python.org/dev/peps/pep-0249/) the `fetchall` method raises an [`Error`](https://www.python.org/dev/peps/pep-0249/#error) and thus sense we should use the SQLAlchemy equivalent exception here rather than `BaseException`.